README.md

```
@@ -1,156 +1,213 @@
-<!-- Header Banner -->
-<!-- Profile README | @rammurmu | AI x Modern Tech x UI/UX -->
+<!-- 🌈 Premium Animated Header -->
+<p align="center">
+  <img src="https://capsule-render.vercel.app/api?type=waving&color=0:0f2027,100:2c5364&height=220&section=header&text=Ram%20Murmu&fontSize=45&fontAlignY=35&animation=fadeIn&fontColor=ffffff"/>
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)